geps/gep-995/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: internal.gateway.networking.k8s.io/v1alpha1
+kind: GEPDetails
+number: 995
+name: Named route rules
+status: Provisional
+authors:
+  - guicassolato
+changelog:
```

Comment:
@youngnick, add relationships to Policy Attachment here, maybe once finishing splitting GEP-713 into [Direct](https://github.com/kubernetes-sigs/gateway-api/issues/2648) and [Inherited](https://github.com/kubernetes-sigs/gateway-api/issues/2649)?

Reply:
Done it.